api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -162,8 +162,18 @@ message HttpConnectionManager {
 
     // Configuration for an external tracing provider.
     // If not specified, Envoy will fall back to using tracing provider configuration
-    // from the bootstrap config.
-    // [#not-implemented-hide:]
+    // from the bootstrap config (for short-term backwards compatibility).
```

Comment:
I would make the new docs read as it should be now vs. referring back to the previous behavior. That can be covered in deprecated.rst.

Reply:
fixed